features/step_definitions/custom_web_steps.rb

```diff
@@ -301,6 +301,6 @@ def take_screenshots_with_login
 end
 
 And /^I active the first hovercard after loading the notifications page$/ do
-  page.should have_css '.notifications .hovercardable'
-  first('.notifications .hovercardable').hover
+  page.should have_css '#notifications-list .hovercardable'
+  first('#notifications-list .hovercardable').hover
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.